airflow/providers/apache/kafka/provider.yaml

```
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-kafka
+name: Kafka
+
+description: |
+  `Kafka  <https://kafka.apache.org/>`__
+versions:
+  - 0.3.0
+
+dependencies:
+  - apache-airflow>=2.2.0
```

All providers now have a minimum Airflow version of 2.3.0.
```suggestion
  - apache-airflow>=2.3.0
```